runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -235,32 +261,60 @@ public int appendObjectState(Stream<Map.Entry> entryStream) {
         ImmutableMap<CheckpointEntry.CheckpointDictKey, String> mdkv =
```

Comment:
On master `mdkv` was copied and passed to `appendObjectState` , but its not used in this patch.

Reply:
@Maithem This is missed out. If you check the method "convertAndAppendCheckpointEntry()", it is using the class variable mkdv successfully.  I do not understand the purpose of creating a shallow copy of mdkv into a local variable for checkpoint entry, when the class variable mdkv could have been used instead. Am I missing something here? For cleaning purpose I should remove this statement.

Reply:
Modified, now each continuation chunk will use the copy of mdkv.